util/cbmem/cbmem.c

```diff
@@ -1590,11 +1606,11 @@ int main(int argc, char** argv)
 
 	parse_cbtable(baseaddr, cb_table_size);
 #else
-	unsigned long long possible_base_addresses[] = { 0, 0xf0000 };
 
 	/* Find and parse coreboot table */
-	for (size_t j = 0; j < ARRAY_SIZE(possible_base_addresses); j++) {
-		if (!parse_cbtable(possible_base_addresses[j], 0))
+	size_t base_addresses_size = address_specified ? 1 : ARRAY_SIZE(possible_base_addresses);
```

Comment:
I'd rather not change the loop below. This may have unforeseen consequences if someone specifies custom size, but not address (some reads have side effects). Maybe do something like this instead:
```
if (address_specified) {
    parse_cbtable(addr, size);
} else {
    <old for loop>
}